### ISSUE TITLE
--live option to sepolicy-inject (already implemented)

### ISSUE THREAD

**Issue by xmikos** [phhusson/Superuser#2]
I have seen in README that you have --live option to sepolicy-inject in your TODO. If you want you can use implementation from my sepolicy-inject fork here: https://github.com/xmikos/setools-android

My setools-android/sepolicy-inject is made buildable with NDK out of the box. I have also implemented support for allowing multiple permissions at once in sepolicy-inject.

Example of usage:

    sepolicy-inject -s vdc -t devpts -c chr_file -p read,write -l

**Comment by phhusson** [comment]
Yup i've seen your sepolicy-inject changes, --live is nothing complicated to implement
Multiple arguments is more annoying to write, and I'd like to have supolicy-compatible syntax (which is different from yours)
Also my sepolicy-inject has some features yours doesn't have (ability to create new domains, to set a domain as non-permissive), and doesn't target only Android (though you have a Makefile in  the lowest directory)

And placeholder copy/paste some code of sepolicy-inject .

So, one thing we could do, is make a new project, which would be both standard linux-buildable and ndk buildable, which could be used as a library into placeholder, and implements one supolicy-like frontend (and another frontend compatible with old sepolicy-inject syntax?)
